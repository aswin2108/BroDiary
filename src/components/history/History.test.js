import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import History from './history.component'

it('Card  rendered correctly', ()=>{
    const mockHistory=[
        {
            id: 2022-10-24,
            entry: 'I enjoying coding',
            possitive: 0.951200125,
            negative: 0.000000005,
            neutral: 0.000001462,
            mixed: 0.1500069
        }
    ];
    const cardComp=renderer.create(
        <MemoryRouter>
            <History props={mockHistory} />
        </MemoryRouter>
    ).toJSON();
    expect(cardComp).toMatchSnapshot();
});