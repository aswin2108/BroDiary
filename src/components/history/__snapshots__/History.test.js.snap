// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card  rendered correctly 1`] = `
<div>
  <button
    className="diary-history-button"
    onClick={[Function]}
  >
    Load Previous
  </button>
  <div
    className="entry-list"
  />
</div>
`;
